zkevm-circuits/src/rlp_circuit_fsm.rs

```diff
@@ -1408,6 +1408,7 @@ impl<F: Field> RlpCircuitConfig<F> {
                 update_state!(meta, cb, state, State::LongList);
 
                 constrain_unchanged_fields!(meta, cb; rlp_table.tx_id, rlp_table.format, tag, tag_next, depth);
+                constrain_unchanged_fields!(meta, cb; rlp_table.tx_id, rlp_table.format, tag, tag_next, depth);
```

Comment:
repeated check?

Reply:
The duplicate row is removed. Cause is likely selecting combination strategy in resolving a previous merge conflict.